ui/src/styles/tokens.css

```
@@ -0,0 +1,192 @@
+/* ============================================================
+   PulseTrader — Design Tokens
+   Ported verbatim (r1.s1.w5) from docs/design/mock/tokens.css in the AI
+   workspace repo. Load order: tokens.css -> shared.css -> <screen>.css
+   (screen sheets do not exist yet -- w3/w4 add them in round 3).
+   ============================================================ */
+
+:root {
+  /* ---------- Type ---------- */
+  --font-ui: -apple-system, BlinkMacSystemFont, "SF Pro Display", "SF Pro Text",
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)